src/Lykke.Numerics.Money/Linq/Money.Average.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lykke.Numerics;
+
+using static Lykke.Linq.ThrowHelper;
+
+namespace Lykke.Linq
```

Comment:
In my mind it should be in another namespace. Maybe Lykke.Numerics[.Money][.Linq].

Reply:
I used the same approach, as MS with System.Linq

Moreover, I switched from Lykke.Numerics.Money to Lykke.Numerics, to avoid Lykke.Numerics.Money.Money and Lykke.Numerics.Monet.UMonet, that looks a little bit weird.

Reply:
Probably, it even will be better to use System.Linq namespace for LINQ extensions, to avoid redundant using

Reply:
+ For Lykke.Numerics.Linq